gateway-service/src/test/java/org/zowe/apiml/gateway/filters/post/PostStoreLoadBalancerCacheFilterTest.java

```diff
@@ -73,9 +72,9 @@ void setUp() {
         class GivenAuthenticationAndInstanceInfo {
 
             @Test
-            void dontAddToCache() {
+            void dontAddToCache_WhenShouldFilter_False() {
```

Comment:
the given_when_then pattern we use would make this named `whenShouldFilterIsFalse_thenDontAddToCache